packages/maven/sentry-servlet/4.1.0.json

```
@@ -0,0 +1,8 @@
+{
+    "name": "io.sentry:sentry-servlet",
+    "canonical": "maven:io.sentry:sentry-servlet",
+    "version": "4.1.0",
+    "package_url": "https://search.maven.org/artifact/io.sentry/sentry-servlet",
+    "repo_url": "https://github.com/getsentry/sentry-java",
+    "main_docs_url": "https://docs.sentry.io/platforms/java/guides/servlet"
```

Either we change to this:

```suggestion
    "main_docs_url": "https://docs.sentry.io/platforms/java/"
```

Or get this done:
getsentry/sentry-docs#3087